src/main/java/com/ithaca/user/User.java

```diff
@@ -0,0 +1,43 @@
+package com.ithaca.user;
+
+import javax.persistence.*;
+
+/**
+ * Created by David on 3/24/16.
+ */
+
+@Entity
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    private String name;
+
+    private String password;
+
+    public User() {
+    }
+
+    public User(String name, String password) {
+        this.name = name;
+        this.password = password;
+    }
+
+    public String getName() {
```

Comment:
just leaving a comment so we can see what it looks like. Obviously you know how to write a getter.